opendevin/runtime/utils/bash_command_splitter.py

```diff
@@ -0,0 +1,181 @@
+import re
+
+
+def split_bash_commands(command_string):
```

Comment:
I think `split_bash_commands` and some following test are target for SSHBOX. I think maybe we will deprecated the SSHBOX in the future. Not sure whether it worth your effort on it.

Reply:
There is at least a new unit test for it.